src/sidebar/components/Annotation/AnnotationQuote.js

```diff
@@ -15,15 +15,15 @@ import StyledText from '../StyledText';
  * @prop {string} quote
  * @prop {boolean} [isFocused]
  * @prop {boolean} [isOrphan]
- * @prop {SidebarSettings} [settings] - Used for theming.
+ * @prop {SidebarSettings} settings
```

Comment:
This module changed because the improvements to `withServices` types surfaced some existing errors here.